CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
 * fix: [#1851](https://github.com/lambdaclass/cairo-vm/pull/1851):
   * Fix unsorted signature and mod builtin outputs in air_private_input.
 
+* feat: add `EvalCircuit` and `TestLessThanOrEqualAddress` hints [#1843](https://github.com/lambdaclass/cairo-vm/pull/1843)
+
```

Comment:
This should be moved to upcoming changes, right?

Reply:
Oh you are right, changing it. Thanks